test/e2e/data/infrastructure-inmemory/main/bases/cluster-with-topology.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: "${CLUSTER_NAME}"
+  namespace: "${NAMESPACE}"
+spec:
+  clusterNetwork:
+    services:
+      cidrBlocks: ${SERVICE_CIDR:=["10.128.0.0/12"]}
+    pods:
+      cidrBlocks: ${POD_CIDR:=["192.168.0.0/16"]}
+    serviceDomain: ${SERVICE_DOMAIN:="cluster.local"}
+  topology:
+    class: in-memory
+    version: ${KUBERNETES_VERSION}
+    controlPlane:
+      metadata:
+        annotations:
+          controlplane.cluster.x-k8s.io/skip-coredns: ""
+          controlplane.cluster.x-k8s.io/skip-kube-proxy: ""
```

Comment:
I would move those to the ClusterClass + add a comment why we do this at the moment